server/lib/genome/importers/base.rb

```diff
@@ -110,11 +116,15 @@ def create_interaction_claim_type(interaction_claim, type)
           type: Genome::Normalizers::InteractionClaimType.name_normalizer(type)
         )
         if claim_type.nil?
-          raise StandardError, "InteractionClaimType with type #{type} does not exist. If this is a valid type, please create its database entry manually before running the importer."
-        end
+          msg = "Unrecognized InteractionClaimType #{type} from #{interaction_claim.inspect}"
+          raise StandardError, msg unless Rails.env == 'development'
```

Comment:
This will work just fine as written, however you can also just do `Rails.env.development?` to get a boolean directly, rather than manually doing an equality check